tensorflow/contrib/distributions/python/kernel_tests/relaxed_bernoulli_test.py

```diff
@@ -94,12 +94,11 @@ def testZeroTemperature(self):
     """If validate_args, raises InvalidArgumentError when temperature is 0."""
     temperature = constant_op.constant(0.0)
     p = constant_op.constant([0.1, 0.4])
-    dist = relaxed_bernoulli.RelaxedBernoulli(temperature, probs=p,
-                                              validate_args=True)
-    with self.cached_session():
-      sample = dist.sample()
-      with self.assertRaises(errors_impl.InvalidArgumentError):
-        sample.eval()
+    with self.assertRaisesWithPredicateMatch(errors_impl.InvalidArgumentError,
+                                             "x > 0 did not hold"):
+      dist = relaxed_bernoulli.RelaxedBernoulli(temperature, probs=p,
```

Comment:
Unused variable 'dist' [unused-variable]

Reply:
Changed `dist` to `_`.